packages/circuit-ui/components/Card/components/Footer/Footer.tsx

```diff
@@ -13,14 +13,14 @@
  * limitations under the License.
  */
 
-import { FC } from 'react';
+import { FC, HTMLAttributes, PropsWithChildren } from 'react';
 import { css } from '@emotion/react';
 
 import styled, { StyleProps } from '../../../../styles/styled';
 
 type Align = 'left' | 'right';
 
-export interface CardFooterProps {
+export interface CardFooterProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
This change shouldn't be necessary if the type annotation is removed from the `CardFooter` component since Emotion merges the HTML attributes of the element that's passed to `styled` and any custom props.

Reply:
Done in [88d33cf](https://github.com/sumup-oss/circuit-ui/pull/1554/commits/88d33cf91563faef62dd0a121ff61dd860d38d22)